### PR TITLE
Move templates and path

### PR DESCRIPTION
**Pull request by joecohens**
Closes https://github.com/AltThree/Badger/issues/4

/cc ping @AltThree/owners

### CODE REVIEW

**Comment by jbrooksuk** [comment]
:+1:

File: src/Render/AbstractRender.php [hunk 1]
```diff
@@ -40,17 +40,19 @@
     /**
      * Create a new svg render instance.
      *
-     * @param \AltThree\Badger\Calculator\TextSizeCalculatorInterface $textSizeCalculator
+     * @param \AltThree\Badger\Calculator\TextSizeCalculatorInterface $textSizeCalculato
+     * @param string                                                  $path
```
Comment (GrahamCampbell) [inline src/Render/AbstractRender.php]: missing |null

Comment (joecohens) [inline src/Render/AbstractRender.php]: Fixed :)

**Comment by jbrooksuk** [comment]
Can we tag this as v2?

**Comment by vinkla** [comment]
:+1:

**Pull request merged**